awx_collection/plugins/doc_fragments/auth.py

```diff
@@ -10,52 +10,52 @@
 
 class ModuleDocFragment(object):
 
-    # Ansible Tower documentation fragment
+    # Ansible Controller documentation fragment
     DOCUMENTATION = r'''
 options:
-  tower_host:
+  controller_host:
     description:
-    - URL to your Tower or AWX instance.
-    - If value not set, will try environment variable C(TOWER_HOST) and then config files
+    - URL to your Controller or AWX instance.
+    - If value not set, will try environment variable C(CONTROLLER_HOST) and then config files
     - If value not specified by any means, the value of C(127.0.0.1) will be used
     type: str
-  tower_username:
+  controller_username:
     description:
-    - Username for your Tower or AWX instance.
-    - If value not set, will try environment variable C(TOWER_USERNAME) and then config files
+    - Username for your Controller or AWX instance.
+    - If value not set, will try environment variable C(CONTROLLER_USERNAME) and then config files
     type: str
-  tower_password:
+  controller_password:
     description:
-    - Password for your Tower or AWX instance.
-    - If value not set, will try environment variable C(TOWER_PASSWORD) and then config files
+    - Password for your Controller or AWX instance.
+    - If value not set, will try environment variable C(CONTROLLER_PASSWORD) and then config files
     type: str
-  tower_oauthtoken:
+  controller_oauthtoken:
     description:
-    - The Tower OAuth token to use.
+    - The Controller OAuth token to use.
     - This value can be in one of two formats.
     - A string which is the token itself. (i.e. bqV5txm97wqJqtkxlMkhQz0pKhRMMX)
-    - A dictionary structure as returned by the tower_token module.
-    - If value not set, will try environment variable C(TOWER_OAUTH_TOKEN) and then config files
+    - A dictionary structure as returned by the controller_token module.
+    - If value not set, will try environment variable C(CONTROLLER_OAUTH_TOKEN) and then config files
     type: raw
     version_added: "3.7"
   validate_certs:
     description:
-    - Whether to allow insecure connections to Tower or AWX.
+    - Whether to allow insecure connections to Controller or AWX.
     - If C(no), SSL certificates will not be validated.
     - This should only be used on personally controlled sites using self-signed certificates.
-    - If value not set, will try environment variable C(TOWER_VERIFY_SSL) and then config files
+    - If value not set, will try environment variable C(CONTROLLER_VERIFY_SSL) and then config files
     type: bool
-    aliases: [ tower_verify_ssl ]
-  tower_config_file:
+    aliases: [ controller_verify_ssl ]
```

Comment:
If you are doing this, then `tower_verify_ssl` should be an alias alongside `controller_verify_ssl`.

Repeat for every other option.

Even options that don't currently have aliases should get a new alias to the old name.